models/src/main/java/com/vimeo/networking2/Transcode.kt

```diff
@@ -1,15 +0,0 @@
-package com.vimeo.networking2
-
-import com.vimeo.networking2.enums.TranscodeType
-
-/**
- * The transcode information for a video upload.
- */
-data class Transcode(
```

Comment:
What is the reason for getting rid of this class? It seems like it was created in part to replace the deprecated `status` property. The replacement is the `upload` property, which does not contain any information about transcoding, which is why the `transcode` property exists on videos as far as I can tell.

Reply:
I had removed it, because it didn't exists in the current `Video` model. I added it back in.